packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -5,29 +5,42 @@
  * them to suit your project as necessary.
  *
  * More information about configuration can be found at:
- * 
+ *
  * https://trufflesuite.com/docs/truffle/reference/configuration
+ * 
+ * Hands-off deployment with Infura
+ * --------------------------------
+ *
+ * Is your application complex and requires lots of transactions to deploy?
+ * Use this appproach to make deployment a breeze 🏖️:
  *
  * To deploy via Infura you'll need a wallet provider (like @truffle/hdwallet-provider)
  * to sign your transactions before they're sent to a remote public node. Infura accounts
- * are available for free at: infura.io/register.
+ * are available for free at 🔍: https://infura.io/register
  *
  * You'll also need a mnemonic - the twelve word phrase the wallet uses to generate
- * public/private key pairs. If you're publishing your code to GitHub make sure you load this
- * phrase from a file you've .gitignored so it doesn't accidentally become public.
- *
+ * public/private key pairs. You can store your secret 🤐 variables in a .env file. In your 
+ * project root, run `$ npm install dotenv`. Create .env (which should be .gitignored) 
+ * and declare your MNEMONIC and infura PROJECT_ID variables inside.
+ * 
+ * Deployment with Truffle Dashboard
+ * ---------------------------------
+ * 
+ * Are you concerned about the security 🤔? Truffle dashboard lets you review transactions in detail, 
+ * and leverages MetaMask for signing, so there's no need to copy-paste your mnemonic.
+ * Please see this link for more details 🔎: https://trufflesuite.com/docs/truffle/getting-started/using-the-truffle-dashboard/
```

Comment:
Could you please make sure that all comments fit under 80 characters per line? (My comment here applies generally; this line is just the one that stood out as longest)